packages/grafana-ui/.storybook/main.ts

```diff
@@ -45,17 +34,18 @@ const mainConfig: StorybookConfig = {
           url: false,
           importLoaders: 2,
         },
+        sassLoaderOptions: {
+          sassOptions: {
+            // silencing these warnings since we're planning to remove sass when angular is gone
+            silenceDeprecations: ['import', 'global-builtin'],
+          },
+        },
       },
     },
     getAbsolutePath('@storybook/addon-storysource'),
     getAbsolutePath('storybook-dark-mode'),
-    getAbsolutePath('@storybook/addon-mdx-gfm'),
```

Comment:
Is gfm supported by default now?

Reply:
Nope. This package is supposed to be a compat package, and we should move to another one, but we don't actually use GFM so I removed it all together.